resolver-dns/src/main/java/io/netty/resolver/dns/UnixResolverDnsServerAddressStreamProvider.java

```diff
@@ -173,10 +174,8 @@ private boolean mayOverrideNameServers() {
                     }
                     if (line.startsWith(NAMESERVER_ROW_LABEL)) {
                         int i = indexOfNonWhiteSpace(line, NAMESERVER_ROW_LABEL.length());
-                        if (i < 0) {
-                            throw new IllegalArgumentException("error parsing label " + NAMESERVER_ROW_LABEL +
-                                    " in file " + etcResolverFile + ". value: " + line);
-                        }
+                        checkPositiveOrZero(i, "error parsing label " + NAMESERVER_ROW_LABEL +
```

Comment:
We should not do any string formatting unless we are throwing the exception. Otherwise we'll do string formatting on the fast path, when there are no errors, just to throw it away.